tests/unit/moduleapi/aclcheck.tcl

```diff
@@ -0,0 +1,66 @@
+set testmodule [file normalize tests/modules/aclcheck.so]
```

Comment:
Is forget to add this test to `runtest-moduleapi`?

Reply:
thanks added